### ISSUE TITLE
Failed to load the existdb package

### ISSUE THREAD

**Issue by dizzzz** [wolfgangmm/atom-existdb#11]
[Enter steps to reproduce:]

1. install on MacOSX, just install plugin

**Atom**: 1.14.0 x64
**Electron**: 1.3.13
**OS**: Mac OS X 10.12.3
**Thrown From**: [existdb](https://github.com/wolfgangmm/atom-existdb) package 0.3.3

### Stack Trace

Failed to load the existdb package

```
At Duplicate key 'ctrl-shift-r'

Error: Duplicate key 'ctrl-shift-r'
    at Object.detectDuplicateKeys (/app.asar/node_modules/season/lib/cson.js:268:13)
    at /app.asar/node_modules/cson-parser/lib/parse.js:115:38
    at Array.reduce (native)
    at Obj (/app.asar/node_modules/cson-parser/lib/parse.js:107:30)
    at transformNode (/app.asar/node_modules/cson-parser/lib/parse.js:197:12)
    at Value (/app.asar/node_modules/cson-parser/lib/parse.js:79:14)
    at transformNode (/app.asar/node_modules/cson-parser/lib/parse.js:197:12)
    at /app.asar/node_modules/cson-parser/lib/parse.js:114:17
    at Array.reduce (native)
    at Obj (/app.asar/node_modules/cson-parser/lib/parse.js:107:30)
    at transformNode (/app.asar/node_modules/cson-parser/lib/parse.js:197:12)
    at Value (/app.asar/node_modules/cson-parser/lib/parse.js:79:14)
    at transformNode (/app.asar/node_modules/cson-parser/lib/parse.js:197:12)
    at Block (/app.asar/node_modules/cson-parser/lib/parse.js:76:14)
    at transformNode (/app.asar/node_modules/cson-parser/lib/parse.js:197:12)
    at Object.parse (/app.asar/node_modules/cson-parser/lib/parse.js:203:12)
    at parseObject (/app.asar/node_modules/season/lib/cson.js:42:23)
    at parseContentsSync (/app.asar/node_modules/season/lib/cson.js:68:16)
    at Object.readFileSync (/app.asar/node_modules/season/lib/cson.js:170:14)
    at /app.asar/src/package.js:439:38
    at Array.map (native)
    at Package.module.exports.Package.loadKeymaps (/app.asar/src/package.js:436:46)
    at /app.asar/src/package.js:114:19
    at Package.module.exports.Package.measure (/app.asar/src/package.js:96:15)
    at Package.module.exports.Package.load (/app.asar/src/package.js:110:12)
    at PackageManager.module.exports.PackageManager.loadPackage (/app.asar/src/package-manager.js:468:14)
    at PackageManager.module.exports.PackageManager.activatePackage (/app.asar/src/package-manager.js:548:30)
    at /app.asar/node_modules/settings-view/lib/package-manager.js:439:29
    at exit (/app.asar/node_modules/settings-view/lib/package-manager.js:69:16)
    at triggerExitCallback (/app.asar/src/buffered-process.js:303:11)
    at /app.asar/src/buffered-process.js:316:11
    at /app.asar/src/buffered-process.js:185:9)
    at emitOne (events.js:101:20)
    at Socket.emit (events.js:188:7)
    at Pipe._handle.close [as _onclose] (net.js:493:12)
```

### Commands

```
```

### Non-Core Packages

```
existdb 0.3.3 
hyperclick 0.0.40 
linter 1.11.21 
```

**Comment by duncdrum** [comment]
same here now with eXist 3.0

**Comment by wolfgangmm** [comment]
Should be fixed with existdb 0.4.1

**Comment by dizzzz** [comment]
thnx!
